src/style.css

```diff
@@ -0,0 +1,60 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf"), format("ttf");
+  src: url("./fonts/Avenir-Heavy.ttf"), format("ttf");
+}
```

Comment:
You should add separate font-face for each font-weight (one for normal, one for bold)